docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
     >>> print("ratio in recall = ", grouped_metric.ratio(method='between_groups'))    
     ratio in recall =  0.0
 
+
+Fairness metrics
+----------------
+
+Parity constraints
+^^^^^^^^^^^^^^^^^^
+In the sections below, we review the most common fairness metrics, as well
+as their underlying assumptions and suggestions for use. Each metric requires
+that some aspects of the predictor behavior be comparable across groups. In 
+the mathematical definitions below, let :math:`X` denote a feature vector 
+used for predictions, :math:`A` be a single sensitive feature (such as age 
+or race), and :math:`Y` be the true label. Parity metrics are phrased in 
+terms of expectations with respect to the distribution over :math:`(X,A,Y)`.
+
+Demographic parity
+~~~~~~~~~~~~~~~~~~
+Demographic parity is also known as *statistical parity*. A classifier
+:math:`h` satisfies demographic parity under a distribution over
```

Comment:
Done! 

> Demographic parity is a fairness metric whose goal is to ensure a machine learning model's predictions are independent of membership in a sensitive group. In other words, demographic parity is achieved when the probability of a certain prediction is not dependent on sensitive group membership. In the binary classification scenario, demographic parity refers to equal selection rates across groups. For example, in the context of a resume screening model, equal selection would mean that the proportion of applicants selected for a job interview should be equal across groups.

[Formulas]

> Failing to achieve demographic parity could generate allocation harms. Allocation harms occur when AI systems allocate 
opportunities, resources, or information differently across different groups (for example, an AI hiring system that is more likely to advance resumes of male applicants than resumes of female applicants regardless of qualification). 
Demographic parity can be used to assess the extent of allocation harms because it reflects an assumption that resources should be allocated proportionally across groups. Of the metrics described in this section, it can be the easiest 
to implement. However, operationalizing fairness using demographic parity rests on a few assumptions: that either the dataset is not a good representation  of what the world actually looks like (e.g., a resume assessment system that is 
more likely to filter out qualified female applicants due to an organizational bias towards male applicants, regardless of skill level), or that the dataset is an accurate representation of the world, but the world is unjust. In reality, this may not be the case. The dataset might be an accurate representation of what the world actually looks like, or the process being modeled may not be unjust.

I'll think through additional examples for the underlying assumptions. They were mentioned in the issue, but I'd appreciate it if you could expand on what you were thinking here a little more. Thank you!